core/vtk/ttkPlanarGraphLayout/ttkPlanarGraphLayout.cpp

```diff
@@ -6,10 +6,15 @@
 #include <vtkCellArray.h>
 #include <vtkFloatArray.h>
 #include <vtkInformation.h>
+#include <vtkInformationVector.h>
 #include <vtkPointData.h>
 #include <vtkSmartPointer.h>
 #include <vtkUnstructuredGrid.h>
 
+#include <FTMTreePPUtils.h>
+#include <ttkFTMTreeUtils.h>
```

Comment:
already included in ttkMergeTreeVisu.h?

Reply:
Actually no, they are not used by ttkMergeTreeVisu.h but only in ttkPlanarGraphLayout.cpp to construct the trees from the vtkUnstructuredGrid and compute the persistence pairs.

Reply:
ok!